src/platform/renesas/KeyValueStoreManagerImpl.cpp

```diff
@@ -17,14 +17,84 @@
  */
 
 #include <platform/KeyValueStoreManager.h>
+#include <unordered_map>
```

Comment:
include standard headers 1st

Reply:
Depends: for zephyr(for example), zephyr/kernel always needs to be included first.

Also what we did at my previous employer, we did system includes last, just as a step for extra validation of dependencies(there might be more reasons).